docs/polkadot/implementors-guide/guide.md

```diff
@@ -0,0 +1,660 @@
+| Process A         |        |       Overseer / Message     |        | Process B         |
+|                   -------->>                  Bus         -------->>                   |
+|                   |        |                              |        |                   |
++-------------------+        |                              |        +-------------------+
+                             |                              |                             
+                             +------------------------------+                             
+```
+
+This communication prevents a certain class of race conditions. When the Overseer determines that it is time for processes to begin working on top of a particular relay-parent, it will dispatch a `StartWork` message to all processes to do so, and those messages will be handled asynchronously by those processes. Some processes will receive those messsages before others, and it is important that a message sent by Process A after receiving `StartWork` message will arrive at Process B after its `StartWork` message. If Process A maintaned an independent channel with Process B to communicate, it would be possible for Process B to handle the side message before the `StartWork` message, but it wouldn't have any logical course of action to take with the side message - leading to it being discarded or improperly handled. Well-architectured state machines should have a single source of inputs, so that is what we do here.
+
+It's important to note that the overseer is not aware of the internals of processes, and this extends to the jobs that they spawn. The overseer isn't aware of the existence or definition of those jobs, and is only aware of the outer processes with which it interacts. This gives process implementations leeway to define internal jobs as they see fit, and to wrap a more complex hierarchy of state machines than having a single layer of jobs for relay-parent-based work.
+
+Futhermore, the protocols by which processes communicate with each other should be well-defined irrespective of the implementation of the process. In other words, their interface should be distinct from their implementation. This will prevent processes from accessing aspects of each other that are beyond the scope of the communication boundary.
+
+---
+
+### Candidate Backing Process
+
+#### Description
+
+The Candidate Backing Process is the process that a validator engages in to contribute to the backing of parachain candidates submitted by other validators.
+
+Its role is to produce backed candidates for inclusion in new relay-chain blocks. It does so by issuing signed [Statements](#Statement-type) and tracking received statements signed by other validators. Once enough statements are received, they can be combined into backing for specific candidates.
+
+It also detects double-vote misbehavior by validators as it imports votes, passing on the misbehavior to the correct reporter and handler.
```

Comment:
You mean double initial proposal of a candidate?  We want parachain backing validators to votes for one anothers' suggestions, but the block producer makes the choice.

Reply:
No, stuff like voting `Valid`/`Invalid` on the same block or seconding two candidates under the same relay-parent.